### DISCUSSION TITLE
Speaker Diarization in Whisper Faster

### DISCUSSION THREAD

**Discussion by Joelson-Forte** [Purfview/whisper-standalone-win#120] [Ideas]
Hi,
Do you think it is possible to add the Speaker Diarization option to Whisper Faster? I think Whisper X and Whisper CPP have this option and in Whisper Open Ai is used the Pyannote [Python], But as I don't use any of them, I'm not completely sure of this information.

https://www.youtube.com/watch?v=OWmkoOQzqC4
https://www.youtube.com/watch?v=Wn2avfPZiSw

**Comment by LokiCode404** [comment]
Yes, I think it is a necessary implementation and that it is an option that should be enabled through a parameter/argument in the execution of Whisper-Faster.
